tsconfig.json

```diff
@@ -1,23 +1,72 @@
+// This file supports comments.
+// Documentation: https://www.typescriptlang.org/tsconfig
 {
     "compilerOptions": {
-        "typeRoots": ["types/pixi.d.ts", "types/p2.d.ts", "types/phaser.d.ts", "types/particlestorm.d.ts"],
-        "lib": ["dom", "es2019", "es2020.string"],
+        // "baseUrl": ".",
+        // "outDir": "./dist/tsc",
+
+        // Library versions
+        "moduleResolution": "node",
         "module": "commonjs",
-        "noImplicitAny": false,
-        "target": "es2017",
+        "target": "ES6",
+
+        // Types
+        "lib": [
```

Comment:
These settings are equivalent to the old ones, correct?

Reply:
Not exactly.  ES6 is ES2015, but the original was 2017.  I called this out in the commit message:  the reason was that ES6 is more-widely supported than ES2017.  But we can change it back to ES2017 if you'd like.

As for `"lib"`:  Using `"ESNext"` means we don't have to list every single deviation from ES6 in the source code.  This was also called-out in the commit message.  ;)